app/assets/javascripts/discourse/app/components/user-stream.js

```diff
@@ -71,6 +73,28 @@ export default Component.extend(LoadMore, {
     this._lastDecoratedElement = lastElement;
   },
 
+  _scrollToLastPosition() {
+    let scrollTo = this.session.userStreamScrollPosition;
+    if (scrollTo && scrollTo >= 0) {
```

Comment:
why do we need to check for scrollTo presence here?

```
null > 0 // false
undefined > 0 // false
```

Reply:
We used the same pattern everywhere so I just copied it. Your comments makes sense though and I replaced the pattern everywhere. :+1: